src/utils/registrable.beneficial.owner.ts

```diff
@@ -47,13 +48,17 @@ export const postRegistrableBeneficialOwner = async (req: Request, res: Response
     setExtraData(req.session, appData);
     await saveAndContinue(req, session, false);
 
-    if (noChangeFlag) {
-      noChangeHandler(req, res);
+    if (isRemoveJourney(req)) {
+      return res.redirect(config.REMOVE_CONFIRM_STATEMENT_URL);
     } else {
```

Comment:
I think that the `if (noChangeFlag)` check can go on this line, with no need for an extra conditional block.

Reply:
Rearranged this and added validation flag for not matching screen  - AC4

Reply:
Nice.